src/components/organisms/SelectionControlsInput/SelectionControlsInput.tsx

```diff
@@ -0,0 +1,146 @@
+import InputWrapper from 'components/molecules/InputWrapper/InputWrapper'
+import BaseButton from 'components/atoms/BaseButton/BaseButton'
+import { ReactComponent as DropdownArrow } from 'assets/icons/dropdown.svg'
+import { useClickAway } from 'ahooks'
+import DropdownContent from 'components/organisms/DropdownContent/DropdownContent'
+
+import classes from './styles.module.scss'
+import { filter, includes, map } from 'lodash'
+
+export enum DropdownSizeTypes {
+  L = 'l',
+  M = 'm',
+  S = 's',
+}
+
+export interface SelectionControlsInputProps {
+  name: string
+  error?: FieldError
+  label?: JSX.Element | string
+  ariaLabel: string
+  value?: string
+  options: {
+    label: string
+    value: string
+  }[]
+  onChange: (value: string | string[]) => void
+  disabled?: boolean
+  placeholder?: string
+  multiple?: boolean
+  helperText?: string
+  buttons?: boolean
+  cancelButtonLabel?: string
+  proceedButtonLabel?: string
+  searchInput?: ReactElement
+  dropdownSize?: DropdownSizeTypes
+  tags?: boolean
+}
+
+const SelectionControlsInput = forwardRef<
+  HTMLInputElement,
+  SelectionControlsInputProps
+>(function SelectionControlsInput(
+  {
+    label,
+    name,
+    ariaLabel,
+    value,
+    error,
+    options,
+    onChange,
+    disabled,
+    placeholder,
+    multiple = false,
+    helperText,
+    buttons = false,
+    cancelButtonLabel,
+    proceedButtonLabel,
+    searchInput,
+    dropdownSize,
+    tags = false,
+  },
+  ref
+) {
+  const [isOpen, setIsOpen] = useState(false)
+
+  const toggleDropdown = () => {
+    setIsOpen(!isOpen)
+  }
+
+  const clickAwayInputRef = useRef(null)
+
+  useClickAway(() => {
+    setIsOpen(false)
+  }, [clickAwayInputRef])
+
+  const selectedOptionObjects = filter(options, (option) =>
+    includes(value, option?.value)
+  )
+
+  const selectedOptionLabels = map(selectedOptionObjects, ({ label }) => label)
+
+  const singleSelectMenuLabel = value ? selectedOptionLabels : placeholder
+
+  const dropdownMenuLabel = multiple ? placeholder : singleSelectMenuLabel
+
+  return (
+    <InputWrapper
+      label={label}
+      name={name}
+      error={error}
+      className={classNames(classes.selectionsContainer)}
+      wrapperSizeClass={classNames(
+        classes.focusedWrapper,
+        classes[dropdownSize || 'l']
+      )}
+      onClick={toggleDropdown}
+      ref={ref}
+      clickAwayInputRef={clickAwayInputRef}
+    >
+      <BaseButton
+        className={classNames(
+          classes.toggleDropdown,
+          disabled && classes.disabledDropdown,
+          classes[dropdownSize || 'l']
+        )}
+        id={name}
+      >
+        <p hidden={!placeholder} className={classes.menuLabel}>
+          {dropdownMenuLabel}
+        </p>
+        <DropdownArrow
+          className={classNames(
+            disabled && classes.disabledDropdownIcon,
+            isOpen && !error && classes.openDropdownIcon
+          )}
+        />
+      </BaseButton>
+
+      <DropdownContent
```

Comment:
**[blocker]**

Go over the list of props here, I see that some stuff passed to DropdownContent here are not actually used inside DropdownContent